CMakeLists.txt

```diff
@@ -394,6 +395,30 @@ IF (BUILD_STATIC_LIBRARY)
 
 ENDIF ()
 
+IF (BUILD_UTILITY)
+
+    SET (UTILITY_NAME "${PROJECT_PACKAGE_NAME}")
+    SET (UTILITY_TARGET "${UTILITY_NAME}.exe")
```

Comment:
Do you have to set this name somewhere `UTILITY_NAME`?

Reply:
This is setting the UTILITY_NAME to `open-space-toolkit-astrodynamics`